applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -277,26 +277,34 @@ static void send_hid_report(const struct hid_report_event *event)
 
 	__ASSERT_NO_MSG(usb_hid->sent_report_id == REPORT_ID_COUNT);
 
+	usb_hid->sent_report_id = event->dyndata.data[0];
```

Comment:
That could be moved above `if (state != USB_STATE_ACTIVE) {` to remove assignment under this condition

Reply:
No it cannot. Line 278 has an assert that verifies nothing was in-flight.

Reply:
please leave as is